Packs/Netmiko/Integrations/Netmiko/Netmiko.yml

```diff
@@ -0,0 +1,306 @@
+      - enterasys
+      - enterasys_ssh
+      - extreme
+      - extreme_ers
+      - extreme_ers_ssh
+      - extreme_exos
+      - extreme_exos_ssh
+      - extreme_exos_telnet
+      - extreme_netiron
+      - extreme_netiron_ssh
+      - extreme_netiron_telnet
+      - extreme_nos
+      - extreme_nos_ssh
+      - extreme_slx
+      - extreme_slx_ssh
+      - extreme_ssh
+      - extreme_telnet
+      - extreme_vdx
+      - extreme_vdx_ssh
+      - extreme_vsp
+      - extreme_vsp_ssh
+      - extreme_wing
+      - extreme_wing_ssh
+      - f5_linux
+      - f5_linux_ssh
+      - f5_ltm
+      - f5_ltm_ssh
+      - f5_tmsh
+      - f5_tmsh_ssh
+      - flexvnf
+      - flexvnf_ssh
+      - fortinet
+      - fortinet_ssh
+      - generic_termserver
+      - generic_termserver_ssh
+      - generic_termserver_telnet
+      - hp_comware
+      - hp_comware_ssh
+      - hp_comware_telnet
+      - hp_procurve
+      - hp_procurve_ssh
+      - hp_procurve_telnet
+      - huawei
+      - huawei_ssh
+      - huawei_telnet
+      - huawei_vrpv8
+      - huawei_vrpv8_ssh
+      - ipinfusion_ocnos
+      - ipinfusion_ocnos_ssh
+      - ipinfusion_ocnos_telnet
+      - juniper
+      - juniper_junos
+      - juniper_junos_ssh
+      - juniper_junos_telnet
+      - juniper_ssh
+      - keymile
+      - keymile_nos
+      - keymile_nos_ssh
+      - keymile_ssh
+      - linux
+      - linux_ssh
+      - mellanox
+      - mellanox_mlnxos
+      - mellanox_mlnxos_ssh
+      - mellanox_ssh
+      - mikrotik_routeros
+      - mikrotik_routeros_ssh
+      - mikrotik_switchos
+      - mikrotik_switchos_ssh
+      - mrv_lx
+      - mrv_lx_ssh
+      - mrv_optiswitch
+      - mrv_optiswitch_ssh
+      - netapp_cdot
+      - netapp_cdot_ssh
+      - netscaler
+      - netscaler_ssh
+      - oneaccess_oneos
+      - oneaccess_oneos_ssh
+      - oneaccess_oneos_telnet
+      - ovs_linux
+      - ovs_linux_ssh
+      - paloalto_panos
+      - paloalto_panos_ssh
+      - paloalto_panos_telnet
+      - pluribus
+      - pluribus_ssh
+      - quanta_mesh
+      - quanta_mesh_ssh
+      - rad_etx
+      - rad_etx_ssh
+      - rad_etx_telnet
+      - ruckus_fastiron
+      - ruckus_fastiron_ssh
+      - ruckus_fastiron_telnet
+      - terminal_server
+      - ubiquiti_edge
+      - ubiquiti_edge_ssh
+      - ubiquiti_edgeswitch
+      - ubiquiti_edgeswitch_ssh
+      - vyatta_vyos
+      - vyatta_vyos_ssh
+      - vyos
+      - vyos_ssh
+    - description: If specified, it will override the username specified in the integration
+        instrance.
+      name: override_username
+    - description: If specified, it will override the password specified in the integration
+        instrance.
+      name: override_password
+    description: Execute commands using Netmiko
+    execution: true
+    name: netmiko-cmds
+    outputs:
+    - contextPath: Netmiko.Command
+      description: The command that was executed.
+      type: string
+    - contextPath: Netmiko.DateTime(UTC)
```

Comment:
Context Keys should not contain characters like `()`, please change it.

Reply:
@ChanochShayner Changed.